services/build.gradle.kts

```diff
@@ -17,10 +17,11 @@ dependencies {
     implementation(project(":common"))
 
     implementation(libs.androidx.core)
-    implementation(libs.androidx.navigation.fragment)
+    implementation(libs.androidx.navigation.runtime.ktx)
```

Comment:
Is this a better dependency? Do we think that users might have the same concern?

Reply:
> Do we think that users might have the same concern?

most likely but this one only has a few transitive dependencies and its mainly the common/runtime bits, compared to the `fragment`, its much more niche/less likely to be an issue